src/renderer/dashboard/dashboard-marketplace-applications.jsx

```diff
@@ -0,0 +1,229 @@
+	const minutes = leadingZero(date.getMinutes());
+	return { year, month, day, hours, minutes };
+};
+
+const RenderDate = ({ timestamp }) => {
+	const { year, month, day } = getAbrDateFromTimestamp(timestamp);
+	return (
+		<>
+			{day} {month} {year}
+		</>
+	);
+};
+
+const ApplicationIcon = ({ classes, title }) => {
+	if (title.toLowerCase().includes('incorporate')) {
+		return (
+			<div className={classes.applicationIcon}>
+				<IncorporationsRoundedIcon />
+			</div>
+		);
+	} else if (title.toLowerCase().includes('bank account')) {
+		return (
+			<div className={classes.applicationIcon}>
+				<BankRoundedIcon />
+			</div>
+		);
+	} else {
+		return (
+			<div className={classes.applicationIcon}>
+				<ExchangeRoundedIcon />
+			</div>
+		);
+	}
+};
+
+const ApplicationsList = ({ classes, route, applications }) => (
+	<>
+		<div className={`${classes.flex} ${classes.flexColumn}`}>
+			{applications.slice(0, 3).map(application => {
+				return (
+					<div
+						key={application.id}
+						className={`${classes.applicationRecord} ${classes.justifySpaceBetween}`}
+					>
+						<div className={classes.flex} style={{ alignItems: 'center' }}>
+							<ApplicationIcon classes={classes} title={application.title} />
+							<div className="application-type">
+								<Typography variant="h6">{application.title}</Typography>
+								<Typography variant="subtitle2" color="secondary">
+									<RenderDate timestamp={application.applicationDate} />
+								</Typography>
+							</div>
+						</div>
+						<div className={classes.applicationStatus}>
+							<HeaderIcon status={application.currentStatus} />
+							<Typography>{application.currentStatusName}</Typography>
+						</div>
+					</div>
+				);
+			})}
+		</div>
+		<div className={`${classes.flex} ${classes.allApplications}`}>
+			<Button onClick={route}>
+				<DropdownIcon width="18px" height="10px" />
+				<Typography variant="overline" className={classes.btnText} color="secondary">
+					View All Applications
+				</Typography>
+			</Button>
+		</div>
+	</>
+);
+
+class DashboardMarketplaceApplications extends PureComponent {
+	marketplaceRoute = () => {
+		this.props.dispatch(push('marketplace'));
+	};
+
+	applicationsRoute = () => {
+		this.props.dispatch(push('/main/selfkeyIdApplications'));
+	};
+
+	render() {
+		const { classes, applications } = this.props;
+		const isEmpty = applications.length < 1;
+		return (
+			<Grid item className={classes.dmaWrap}>
+				<Typography variant="h1" className={classes.title}>
+					Marketplace Applications
+				</Typography>
+				{isEmpty && isEmpty ? (
+					<EmptyState classes={this.props.classes} route={this.marketplaceRoute} />
+				) : (
+					<ApplicationsList
+						classes={this.props.classes}
+						route={this.applicationsRoute}
+						applications={applications}
+					/>
+				)}
+			</Grid>
+		);
+	}
+}
+
+const mapStateToProps = state => {
+	return {
+		applications: kycSelectors.selectApplications(state)
```

Comment:
I'm not sure about this, I think this needs a 
`await dispatch(kycOperations.loadApplicationsOperation()); `
to show the applications

Reply:
Good point.